test/api/test_tools.py

```diff
@@ -436,6 +439,13 @@ def test_run_by_versions(self):
             output1_content = self.dataset_populator.get_history_dataset_content(history_id, dataset=output1)
             self.assertEqual(output1_content.strip(), "Version " + version)
 
+    @skip_without_tool("multiple_versions")
+    @uses_test_history(require_new=False)
```

Comment:
You need to add `uses_test_history` to the imports.

Reply:
That didn't exist in 18.05, but I can jjust use the old style populator

Reply:
Uh, I don't even need a history here